webdriver-spec.html

```diff
@@ -4078,20 +4078,29 @@
  </tr>
 </table>
 
-<p>The <dfn>Find Element</dfn> <a>command</a> is used
-  to find an element in the <a>current browsing context</a> that can be used for
-  future <a>commands</a>.
+<p>The <dfn>Find Element</dfn> <a>command</a>
+ is used to find an <a>element</a> in the <a>current browsing context</a>
+ that can be used for future <a>commands</a>.
 
 <p>The <a>remote end steps</a> are:
 
 <ol>
- <li><p>Let <var>all</var> be equal to false.
+ <li><p>Let <var>all</var> be false.
+
+ <li><p>Let <var>start node</var> be
+  the <a>current browsing context</a>’s <a>document element</a>.
+
+ <li><p>Let <var>location strategy</var> be the result
+  of <a>getting a property</a> called "<code>using</code>".
 
- <li><p>Let <var>start node </var> be <a><code>html</code> element</a>.
+ <li><p>If <var>location strategy</var> is <a>undefined</a>,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
 
- <li><p>Let <var>location strategy</var> be the result of <a>getting a property</a> called "<code>using</code>".
+ <li><p>Let <var>selector</var> be the result
+  of <a>getting a property</a> called "<code>value</code>".
 
- <li><p>Let <var>selector</var> be the result of <a>getting a property</a> called "<code>value</code>".
+ <li><p>If <var>selector</var> is <a>undefined</a>,
+  return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
```

Comment:
The patch in #432 checks for an empty string and null, not that they are missing.